src/tags/object/Image.js

```diff
@@ -297,10 +297,22 @@ const Model = types.model({
 
   // @todo the name is for backward compatibility; change the name later
   get _value() {
-    const value = self.parsedValue;
+    let value = self.parsedValue;
+
+    if (Array.isArray(value)) value = value[self.currentImage];
 
-    if (Array.isArray(value)) return value[self.currentImage];
-    return value;
+    if (!self.imageCrossOrigin) {
+      return value;
+    } else if (isFF(FF_DEV_4081) && /^https?:.*?s3\.amazonaws\.com\//g.test(value)){
```

Comment:
Hey @BradNeuberg 👋 . There is one other issue I came across with regards to this check and it occurs if the URL is a presigned URL. This query param addition made like this will break any presigned URL. If the URL is an S3 URL and it does not have an existing query param present `X-Amz-Signature`, then this cache bust would be fine.

Reply:
Thanks @bmartel, good catch. I don't have an AWS S3 environment setup for testing (I use Google Cloud at Planet for my development). Do you have an example presigned S3 URL you can provide for testing (you can change the key on it to some gibberish so you don't leak anything)? It sounds like I should update my regular expression to handle the S3 presigned URL case?

Reply:
Actually this is a tricky issue you've identified. So if I have an signed S3 URL like the following:

https://somebucket.s3.amazonaws.com/some-object-key?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIOSFODNN7EXAMPLE%2F20230329%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230329T123456Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=abcdefghijklm1234567890EXAMPLE

Can I simply not add a cache bust parameter at the end? Will that invalidate the entire S3 signed URL? Note that doing the cache bust works for Google Cloud signed bucket URLs, which is what we use at Planet. Not sure if there's something subtly different about S3 signed ones.

Reply:
Probably a safer way to do this cache bust would be to check if it is an amazon s3 url and whether it is a signed url, then perform the cache bust.

```suggestion
    } else if (isFF(FF_DEV_4081) && /^https?:.*?s3\.amazonaws\.com\//g.test(value) && !(new URL(value)).searchParams.has('X-Amz-Signature')) {
```

Reply:
Thanks @bmartel, I've incorporated your code suggestion and pushed it back to this PR. Note that I am unable to test signed AWS URLs.